frontend/app/src/components/home/Home.svelte

```diff
@@ -544,7 +544,10 @@
     function addMembers() {
         if ($selectedChatId !== undefined) {
             rightPanelHistory.update((history) => {
-                return [...history, { kind: "add_members" }];
+                return [
+                    ...history.filter((p) => p.kind !== "add_members"),
```

Comment:
Do we really want to clear all of the matching historical entries?
I thought the issue was that it was adding lots of the same entries in a row so we could just skip adding if the latest entry is of type 'add_members'?